frontends/web/src/components/amount/conversion-amount.tsx

```diff
@@ -54,7 +54,7 @@ export const ConversionAmount = ({
 
   return (
     <span className={styles.txConversionAmount}>
-      {conversion && amountToShow ? (
+      {(conversion || sendToSelf) && amountToShow ? (
```

Comment:
If this is a `sendToSelf`, we do not need `conversion` as we show the coin amount, and not its conversion into a fiat value.